addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -450,6 +450,22 @@ var SnippetEditor = Widget.extend({
 
         return Promise.all(defs);
     },
+    /**
+     * Returns true if the element matches a snippet block that cannot be
+     * droppped in a sanitized HTML field.
+     *
+     * @param {jQuery} $el
+     * @return {boolean}
+     */
+    _isForbidSanitize($el) {
+        const $scroll = $('#o_scroll');
+        for (const cl of $el[0].classList) {
+            if ($scroll.find('.oe_snippet_body.' + cl).closest('.oe_snippet').find('.oe_forbid_sanitize').length) {
```

Comment:
Yep, it's annoying in 13.0 :grimacing: But this seems not robust enough, grep this: `Associate in-page snippets to their name` there is some code which is probably more robust (if it works... did it 5 years ago... :grimacing:) to search a snippet from its classes. From 14.0, you will just be able to use the `data-snippet` attribute on snippets to do this search (we'll change that in forward-port).

Reply:
I tried to reuse it, but it makes either too many things match of too few - because of attributes such as `s_parallax_is_fixed`.
I therefore rewrote something similar - explicitly excluding the `s_parallax_...` classes.